app/experimenter/urls.py

```diff
@@ -10,6 +10,9 @@
     re_path(r"^api/v2/experiments/", include("experimenter.experiments.api.v2.urls")),
     re_path(r"^api/v3/", include("experimenter.experiments.api.v3.urls")),
     re_path(r"^api/v4/", include("experimenter.experiments.api.v4.urls")),
+    re_path(
+        r"^api/visualizations/v1/", include("experimenter.visualization.api.v1.urls")
```

Comment:
I think this breaks the API versioning pattern we've been using.  We've been using the V1 API as the public read only API but since this is sensitive data I'd expect it to be behind an auth gate.  The V3 API is auth gated and currently hosts stuff used by the nimbus frontend, so you could stick it in there, or make a new one and call it V5?

Also in general the pattern we've been using is

`api/Vx/<resource>/`

so I'd expect it to something like that?

Reply:
I put it under v3